src/utils/getPeopleByName.ts

```diff
@@ -0,0 +1,8 @@
+import { Person } from '../types';
+
+export const getPersonByName = (
+  name: string | null,
+  people: Person[],
+): Person | undefined => {
+  return people.find(person => person.name === name);
```

Comment:
Consider making your comparison case-insensitive to enhance user experience. You can convert both `person.name` and `name` to lowercase before comparing them. This way, the function will return a match even if the user types the name in a different case. Here is a suggested change: `person.name.toLowerCase() === name?.toLowerCase()`.